tomland.cabal

```diff
@@ -26,13 +26,16 @@ library
 
   build-depends:       base >= 4.9 && < 5
                      , hashable
+                     , megaparsec
+                     , parser-combinators
```

Comment:
Why is this needed?

Reply:
`parser-combinators` is a dependency for `megaparsec`. Some parser combinators are not exported by `megaparsec`. Instead more general versions of them are in the `parser-combinators` packages. Specifically, I need these ones:

```haskell
sepBy1   :: MonadPlus m => m a -> m sep -> m (NonEmpty a)
between  :: Applicative m => m open -> m close -> m a -> m a
sepBy    :: Alternative m => m a -> m sep -> m [a]
manyTill :: Alternative m => m a -> m end -> m [a]
```